docs/architecture/architecture-overview.adoc

```diff
@@ -0,0 +1,244 @@
+     Short Names:
+       mc
+     Singular:  mobileclient
+   Scope:       Namespaced
+   Version:     v1alpha1
+ Status:
+   Accepted Names:
+     Kind:       MobileClient
+     List Kind:  MobileClientList
+     Plural:     mobileclients
+     Short Names:
+       mc
+     Singular:  mobileclient
+   Conditions:
+     Last Transition Time:  <nil>
+     Message:               no conflicts found
+     Reason:                NoConflicts
+     Status:                True
+     Type:                  NamesAccepted
+     Last Transition Time:  2018-01-04T09:56:32Z
+     Message:               the initial names have been accepted
+     Reason:                InitialNamesAccepted
+     Status:                True
+     Type:                  Established
+ Events:                    <none>
+ MacBook-Pro:architecture user1$
+====
+
+* *Step 4*: Install the Mobile CLI
+
+The functionality provided through the OpenShift UI is also available on the command line. MC extends the Kubernetes and OpenShift CLI to provide mobile specific commands. More details in how to extend the base CLI is available https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/[here]
+
+All of the functionality available via the UI will be available from the CLI. The mobile CLI runs standalone aswell as through the _kubectl_ and _oc_ commands. E.g.
+
+ MacBook-Pro:images joeBloggs$ mobile --help
+ A brief description of your application
+ Usage:
+  mobile [command]
+ Available Commands:
+  create      create clients integrations etc...
+  delete      delete clients, clientbuilds etc
+  get         get clients, service and clientbuilds
+  help        Help about any command
+  start       start clientbuild
+  stop        stop clientbuild
+ Flags:
+  -h, --help               help for mobile
+      --namespace string   --namespace=myproject
+  -o, --output string      -o=json -o=template (default "table")
+ Use "mobile [command] --help" for more information about a command.
+ MacBook-Pro:images joeBloggs$
+
+There is a separate repo for the Mobile CLI. Setup and usage guides are available https://github.com/aerogear/mobile-cli[here].
+
+ Note; currently the Mobile CLI is not installed during MC startup, but the intention is to add it to the installation process in the future.
+
+Once installed, there is no live/running MC process/server. There is no openshift pod for MC. Once MC is installed and configured its functionality is available throughout all Openshift projects/namespaces.
+
+== MC Functionality
+=== Create a representation of a Mobile App
+The MC UI extension provides the user with the capabilitiy to create a representation of a Mobile App inside their Openshift project.
+
+MC allows the user to select the Mobile filter in the Service Catalog and select a representation of an App type to create. Note; an App representation is a set of basic properties for the App. It is not a running process. Currently its a visual represenation of an App.
```

Comment:
We can also add info what mobile apps are for. For example:
`The mobile app is a resource that represents your mobile client app as part of the OpenShift UI. It gives us the context and information needed to show you relevant information around your particular mobile runtime as well as allowing us setup the different kind of mobile client builds required.`

From: https://github.com/aerogear/mobile-cli